src/writer.js

```diff
@@ -33,6 +39,7 @@ class Writer {
       if (this.length < this._size) {
         this._queue.push(buffer)
       } else {
+        log.error(() => `Writer buffer full at ${this._size} entries. Squeezing trace in place of another`)
```

Comment:
Unrelated to current feature. Happy to accept more PRs to improve our logging though!

Reply:
Whoops, will split this out :+1: